main/src/com/google/refine/model/Cell.java

```diff
@@ -124,12 +124,11 @@ public Object getValue() {
                 return ((Float)value).toString();
             } else if (value instanceof Boolean || value instanceof Number){
                 return value;
-            } else {
-                return value.toString();
-            }
-        } else {
-           return null;
+            } 
+            return value.toString();   
         }
+        return null;
+        
```

Comment:
Those changes are valid but it is really a matter of preference, so unless we have a documented guideline around this, I would just keep it as it is, for simplicity.